### DISCUSSION TITLE
Backup minIO servers

### DISCUSSION THREAD

**Discussion by smateluna** [minio/minio#16965] [Q&A] [answered]
Hello everyone, I have 2 nodes minIO cluster, running over VMware VMs and we need to backup them all or the data.

How can I backup those vm servers in a supported way? We currently backup our VMs using Veeam Backup.

Can I use Veeam to backup minIO server, which uses only one disk for data and OS? It can be restored and run without issues?

I've searched and found MC MIRROR command, but I want to know if backups can be made at VM level, not API level. All the data stored in our minIO servers are velero kubernetes backups, so it doesn't change too often.

Thanks in advance!

Regards.
Sergio.

**Comment by klauspost** [comment, marked as answer]
Single Node Single Drive can be backed up with traditional means.

I suggest you test it out, though. It is not something we test for.
